src/main/java/com/hivemq/cli/mqtt/AbstractMqttClientExecutor.java

```diff
@@ -200,10 +185,15 @@ public boolean isConnected(final @NotNull Subscribe subscriber) {
 
     public @NotNull MqttClient connect(final @NotNull Connect connect) {
 
-
         LoggingContext.put("identifier", "CLIENT " + connect.getIdentifier());
 
-        clientCache.setVerbose(connect.isVerbose());
+        if (isConnected(connect)) {
+            if (connect.isVerbose()) {
+                Logger.trace("Client with key {} is already connected", connect.getKey());
```

Comment:
Maybe there is a prettier way to display this information so that users quickly understand.